src/SerialPort.ts

```diff
@@ -73,14 +74,17 @@ class SerialPort {
    * @param {listener} listener
    * @returns {Subscription} subscription
    */
-  onReceived(listener: (data: Buffer) => void): EventSubscription {
+  onReceived(listener: (buffer: Buffer, path: string) => void): EventSubscription {
```

Comment:
I think it's not necessary to receive path.
```suggestion
  onReceived(listener: (buffer: Buffer) => void): EventSubscription {
```